### PR TITLE
feat: Integration testing without wiremock, cleaning and CAPEX testing

### PR DESCRIPTION
**Pull request by arnaugarcia**
### Summary

* Replaced the Wiremock dependency with OKHTTP3 official one.
* Added tests for the CAPEX order details
* Some cleaning

### Testing

Tested with OkHTTP and tried to do many unit testing as possible but (only the CAPEX part)

### CODE REVIEW

File: src/main/java/com/evpassport/payments/integration/salesforce/SalesforceService.java [hunk 1]
```diff
@@ -24,7 +24,7 @@ public SalesforceLocationInfoDTO getLocationById(String id) {
         Opportunity opportunity = location.getOpportunities()
                 .stream()
                 .findFirst()
-                .orElseThrow();
+                .orElseThrow(); // TODO: Throw a custom exception
```
Comment (arnaugarcia) [inline src/main/java/com/evpassport/payments/integration/salesforce/SalesforceService.java:27]: Yes, I know... but it's the part of the Salesforce integration that we need more info to continue. I just added a TODO to remind myself to fix that when we have news.

File: src/main/java/com/evpassport/payments/repository/PayoutRepository.java
Comment (ev-adias) [inline src/main/java/com/evpassport/payments/repository/PayoutRepository.java:1]: can't you delete the whole class?

File: src/main/java/com/evpassport/payments/service/PayoutService.java
Comment (ev-adias) [inline src/main/java/com/evpassport/payments/service/PayoutService.java:1]: Do we still use this?

Comment (arnaugarcia) [reply src/main/java/com/evpassport/payments/service/PayoutService.java:1]: Done!

File: src/main/java/com/evpassport/payments/repository/PayoutRepository.java
[the remark this answers is not in the document]
Comment (arnaugarcia) [reply src/main/java/com/evpassport/payments/repository/PayoutRepository.java:1]: Yes, I've deleted all the payouts logic now, my bad :(

File: build.gradle [hunk 2]
```diff
@@ -50,20 +50,24 @@ dependencies {
...
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation 'com.squareup.okhttp3:mockwebserver:4.11.0'
     testImplementation 'io.projectreactor:reactor-test'
     testImplementation 'io.rest-assured:rest-assured:5.3.2'
     testImplementation 'org.testcontainers:postgresql'
     testImplementation 'org.testcontainers:junit-jupiter'
 
+    testCompileOnly 'org.projectlombok:lombok:1.18.30'
+
+    testAnnotationProcessor 'org.projectlombok:lombok:1.18.30'
```
Comment (richardjfleming) [inline build.gradle:69]: You could maybe extract these versions to a global version property

ext {
    set('lombokVersion', "1.18.30")
}

**Review by richardjfleming — COMMENTED** [review]
My review is basically meaningless haha without the context but I don't see anything glaringly obvious

File: build.gradle (hunk 2 above)
Comment (arnaugarcia) [reply build.gradle]: Done, thanks for the tip!

**Review by ev-adias — APPROVED** [review]

**Pull request merged**